### ISSUE TITLE
binding prefixes has no effect

### ISSUE THREAD

**Issue by KonradHoeffner** [oxigraph/oxrdflib#10]
Using oxrdflib 0.3.0 with rdflib 6.1.1 and the Oxigraph store, graph.namespace_manager.bind does not cause the prefix to be recognized. The error does not occur when using standard rdflib via `rdflib.Graph()` instead.

## Example

```
import rdflib
import oxrdflib

DATA = """
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
<test> rdfs:label "test".
"""
QUERY = """
SELECT ?label {rdfs:test rdfs:label ?label.}
"""
graph = rdflib.Graph(store="Oxigraph")
graph.namespace_manager.bind('rdfs', rdflib.namespace.RDFS, override=True)
graph.parse(data=DATA,format="ttl")
graph.query(QUERY)
```

## Error

```
$ python test.py
Traceback (most recent call last):
  File "/home/konrad/tmp/test.py", line 16, in <module>
    graph.query(QUERY)
  File "/home/konrad/.local/lib/python3.10/site-packages/rdflib/graph.py", line 1311, in query
    return self.store.query(
  File "/home/konrad/.local/lib/python3.10/site-packages/oxrdflib/__init__.py", line 78, in query
    result = self._inner.query(
SyntaxError: error at 2:25: expected one of Prefix not found, ['%'], ['-' | '0' ..= '9' | '\u{00B7}' | '\u{0300}' ..= '\u{036F}' | '\u{203F}' ..=
 '\u{2040}'], ['.'], [':'], ['A' ..= 'Z' | 'a' ..= 'z' | '\u{00C0}' ..= '\u{00D6}' | '\u{00D8}' ..=
 '\u{00F6}' | '\u{00F8}' ..= '\u{02FF}' | '\u{0370}' ..= '\u{037D}' |
 '\u{037F}' ..= '\u{1FFF}' | '\u{200C}' ..= '\u{200D}' | '\u{2070}' ..=
 '\u{218F}' | '\u{2C00}' ..= '\u{2FEF}' | '\u{3001}' ..= '\u{D7FF}' |
 '\u{F900}' ..= '\u{FDCF}' | '\u{FDF0}' ..= '\u{FFFD}'], ['\\'], ['_']
```

**Comment by Tpt** [comment]
Sorry for this limitation. I have just pushed fc09668380edccab5a2e5d9f297275d5bb7d3b88 that should add this feature. I hope to cut a release in the next few days.

**Comment by Tpt** [comment]
This should be fixed as part of the new v0.3.1 release. Feel free to open again this issue if it keeps not working.
